From 7322c2bde34b5207feb01ee9e8235b13107df3af Mon Sep 17 00:00:00 2001
From: Vishal Sagar <vishal.sagar@xilinx.com>
Date: Wed, 27 Feb 2019 10:15:37 -0800
Subject: [PATCH 0513/1851] dt-bindings: display: xlnx: Add clock framework
 support to DSI Tx

commit cb17f77c7b7183506664a1a7ec5cf8be8c2b0f77 from
https://github.com/Xilinx/linux-xlnx.git

This patch adds support for clock framework in DSI Tx driver by adding
new clock properties for video and DPHY clock.

This breaks backward compatibility as compatible string is not changed.

Signed-off-by: Vishal Sagar <vishal.sagar@xilinx.com>
Reviewed-by: Hyun Kwon <hyun.kwon@xilinx.com>
State: pending
Signed-off-by: Yaliang Wang <Yaliang.Wang@windriver.com>
---
 .../devicetree/bindings/display/xlnx/xlnx,dsi.txt          | 7 +++++++
 1 file changed, 7 insertions(+)

diff --git a/Documentation/devicetree/bindings/display/xlnx/xlnx,dsi.txt b/Documentation/devicetree/bindings/display/xlnx/xlnx,dsi.txt
index 0079c4c6391c..55508167e606 100644
--- a/Documentation/devicetree/bindings/display/xlnx/xlnx,dsi.txt
+++ b/Documentation/devicetree/bindings/display/xlnx/xlnx,dsi.txt
@@ -32,6 +32,11 @@ Required properties:
    value or has to be programmed to the same value that DSI Tx controller is
    configured to.
 
+ - clocks: List of phandles to Video and 200Mhz DPHY clocks.
+
+ - clock-names: Must contain "s_axis_aclk" and "dphy_clk_200M" in same order as
+   clocks listed in clocks property.
+
 Required simple_panel properties:
  - compatible: Value should be one of the panel names in
    Documentation/devicetree/bindings/display/panel/. e.g. "auo,b101uan01".
@@ -53,6 +58,8 @@ Example:
 		#address-cells = <1>;
 		#size-cells = <0>;
 		xlnx,vpss = <&v_proc_ss_0>;
+		clock-names = "dphy_clk_200M", "s_axis_aclk";
+		clocks = <&misc_clk_0>, <&misc_clk_1>;
 		encoder_dsi_port: port@0 {
 			reg = <0>;
 			dsi_encoder: endpoint {
-- 
2.31.1

